storm-core/src/jvm/org/apache/storm/trident/Stream.java

```diff
@@ -57,6 +57,23 @@
 import org.apache.storm.trident.state.StateUpdater;
 import org.apache.storm.trident.util.TridentUtils;
 
+/**
+ * A Stream represents the core data model in Trident, and can be thought of as a "stream" of tuples that are processed
+ * as a series of small batches. A stream is partitioned accross the nodes in the in the cluster, and operations are
```

Comment:
`the nodes in the *in the* cluster` the second `in the` should probably go away.